### PR TITLE
Test-Verify RWO volumes are exclusively mounted by adding replica 2 i…

### PR DESCRIPTION
**Pull request by jilju**
…n pod dc

OCS-896 - RBD: Verify RWO volumes are exclusively mounted by adding replicas - 2 in pod dc
OCS-897 - CephFS: Verify RWO volumes are exclusively mounted by adding replicas - 2 in pod dc

Signed-off-by: Jilju Joy <jijoy@redhat.com>

### CODE REVIEW

File: tests/manage/pv_services/test_verify_rwo_using_dc_pod.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                break
+
+        # Verify the other pod is not coming up Running
+        try:
+            self.pod_not_running = next(pods_iter)
+            wait_for_resource_state(
+                resource=self.pod_not_running, state=constants.STATUS_RUNNING,
+                timeout=10
+            )
+            raise UnexpectedBehaviour(
```
Comment (ebenahar) [inline tests/manage/pv_services/test_verify_rwo_using_dc_pod.py:101]: it seems that this exception will be thrown in any case. Could you please clarify?

Comment (jilju) [reply tests/manage/pv_services/test_verify_rwo_using_dc_pod.py:101]: Expecting ResourceWrongStatusException from wait_for_resource_state.
If that exception did not occur, it is considered as failure and  UnexpectedBehaviour .

In other words, in the normal flow as expected by the test case, line 101 won't be executed.

Comment (ebenahar) [reply tests/manage/pv_services/test_verify_rwo_using_dc_pod.py:101]: got you :)
Please add this explanation to comment in the code

Comment (jilju) [reply tests/manage/pv_services/test_verify_rwo_using_dc_pod.py:103]: Done

**Pull request merged**